packages/color-loupe/README.md

```diff
@@ -32,3 +37,139 @@ import { ColorLoupe } from '@spectrum-web-components/color-loupe';
+<div
+    style="display: flex; flex-direction: row; justify-content: space-between; align-items: flex-start; width: 100%; max-width: 600px;"
+>
+    <!-- Yellow color loupe -->
+    <div style="padding: 100px 0 0; position: relative; min-width: 120px;">
+        <div style="position: relative;">
+            <sp-color-loupe color="yellow" open dir="ltr"></sp-color-loupe>
+        </div>
+    </div>
+
+    <!-- Red color loupe -->
+    <div style="padding: 100px 0 0; position: relative; min-width: 120px;">
+        <div style="position: relative;">
+            <sp-color-loupe color="#ff0000" open dir="ltr"></sp-color-loupe>
+        </div>
+    </div>
+
+    <!-- Blue color loupe -->
+    <div style="padding: 100px 0 0; position: relative; min-width: 120px;">
+        <div style="position: relative;">
+            <sp-color-loupe
+                color="rgba(44, 62, 224, 0.81)"
+                open
+                dir="ltr"
+            ></sp-color-loupe>
+        </div>
+    </div>
+
+    <!-- Green color loupe -->
+    <div style="padding: 100px 0 0; position: relative; min-width: 120px;">
+        <div style="position: relative;">
+            <sp-color-loupe
+                color="hsl(111, 82%, 56%)"
+                open
+                dir="ltr"
+            ></sp-color-loupe>
+        </div>
+    </div>
+</div>
+```
+
+### States
+
+#### Open
+
+The `open` attribute controls whether the loupe is visible. When `open` is present, the loupe displays the color preview.
+
+```html
+<div style="display: flex; flex-direction: row; gap: 20px;">
+    <!-- Loupe is visible -->
+    <div style="padding: 100px 0 0; margin-left:20%">
+        <div style="position:relative">
+            <sp-color-loupe open="" dir="ltr"></sp-color-loupe>
+            <p id="color-context" style="margin-top: 40px">
+                This loupe above this text is visible.
+            </p>
+        </div>
+    </div>
+
+    <!-- Loupe is hidden -->
+    <div style="padding: 100px 0 0;">
+        <div style="position:relative">
+            <sp-color-loupe dir="ltr"></sp-color-loupe>
+            <p id="color-context" style="margin-top: 40px">
+                This loupe above this text is not visible.
+            </p>
+        </div>
+    </div>
+</div>
+```
+
+### Behaviors
+
+The color loupe is typically managed by its parent color component (such as `<sp-color-area>`, `<sp-color-slider>`, or `<sp-color-wheel>`). The loupe automatically appears when the user interacts with the parent component and disappears when the interaction ends.
+
+#### Automatic behavior
+
+- **Touch input**: The loupe automatically appears to prevent the finger from obscuring the selected color
+- **Mouse/Stylus input**: The loupe remains hidden by default for precision pointing devices
+- **Parent control**: The loupe's visibility is managed by the parent color component
+
+### Accessibility
+
+The `<sp-color-loupe>` is designed to work as part of accessible color selection components:
```

Comment:
Can we add an example here of a color loupe inside a selection component, so that consumers can try it and so that they know how to implement it?